docs/source/markdown/options/network.md

```diff
@@ -75,5 +75,4 @@ Valid _mode_ values are:
         host, using the loopback interface instead of the tap interface for improved
         performance
 
-    NOTE: For backward compatibility reasons, if you have a network named `pasta`,
-    Podman will use this instead of the pasta mode.
+    NOTE: For backward compatibility reasons, Podman will use networks named `pasta` instead of the pasta mode.
```

Comment:
```suggestion
    NOTE: For backward compatibility reasons, if a network named `pasta` exists on the container, Podman will use it instead of the pasta mode.
```